grumphp.yml.dist

```diff
@@ -28,9 +36,13 @@ grumphp:
       max_subject_width: 0
 
 parameters:
-  tasks.phpcs.standard: ./vendor/openeuropa/code-review/resources/library-phpcs-ruleset.xml
+  tasks.phpcs.standard: ./vendor/squizlabs/php_codesniffer/src/Standards/PSR12/ruleset.xml
   tasks.phpcs.warning_severity: 0
+  tasks.phpcs.whitelist_patterns: []
   tasks.phpcs.ignore_patterns:
     - vendor/
   tasks.phpcs.triggered_by:
-    - php
\ No newline at end of file
+    - php
+  tasks.phpstan.level: 5
```

Comment:
I set the initial level to 5, it would be reasonable to raise it later and fix remaining issues

Reply:
lets bump it up to level 6 to catch the hint types